test/gtest/ucp/test_ucp_am.cc

```diff
@@ -622,21 +622,22 @@ UCS_TEST_P(test_ucp_am_nbx, rx_rts_reply_am_on_closed_ep, "RNDV_THRESH=32K")
 
 UCS_TEST_P(test_ucp_am_nbx, rx_persistent_data)
 {
-    void *data_desc = NULL;
-    char data       = 'd';
+    void *rx_data = NULL;
+    char data     = 'd';
 
-    set_am_data_handler(receiver(), TEST_AM_NBX_ID, am_data_hold_cb, &data_desc,
+    set_am_data_handler(receiver(), TEST_AM_NBX_ID, am_data_hold_cb, &rx_data,
                         UCP_AM_FLAG_PERSISTENT_DATA);
 
     ucp_request_param_t param;
 
     param.op_attr_mask    = 0ul;
     ucs_status_ptr_t sptr = ucp_am_send_nbx(sender().ep(), TEST_AM_NBX_ID, NULL,
                                             0ul, &data, sizeof(data), &param);
-    wait_for_flag(&data_desc);
-    EXPECT_TRUE(data_desc != NULL);
+    wait_for_flag(&rx_data);
+    EXPECT_TRUE(rx_data != NULL);
+    EXPECT_EQ(data, *(reinterpret_cast<char*>(rx_data)));
```

Comment:
minor: extra ( ) around reinterpret_cast

Reply:
@brminich wdyt?

Reply:
ok, can I fix and squash or you have more comments?

Reply:
yes, i have no more comments